### DISCUSSION TITLE
HP ProBook 4530s

### DISCUSSION THREAD

**Discussion by amervelic** [chris1111/Legacy-Video-patch#29] [General]
pretty bad experience with video legacy patch. after a lot of effort and hours spent, it worked for a while and now after installing security patch 2021 for Mojave 10.14.6, the boot goes correctly without errors but when the login should appear - it does not appear even after a few hours! The only thing that works well is High Sierra 10.13.6 (OC 0.6.6), Mojave worked and Catalina doesn't work, kernel panic after installing Legacy Video Patch.

**Comment by chris1111** [comment]
Follow this for Legacy video patch on 6 series laptop
https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator/issues/57#issuecomment-772659030
anyway your discussion is an issue

**Reply by amervelic** [reply]
I did it ! install Mojave 10.14.6 + security update 2021 (without OC package creator) QE / CI (after clean NVRAM wont work), Wifi, Ethernet, Audio, Bluetooth etc works - Open Core 0.6.7. Please post how you did it with Catalina (OC only) because it doesn't work for me at all (kernel panic).
